src/Microsoft.Identity.Web.Certificate/SignedAssertionFilePathCredentialsLoader.cs

```diff
@@ -36,7 +36,7 @@ public async Task LoadIfNeededAsync(CredentialDescription credentialDescription,
                 {
                     // Given that managed identity can be not available locally, we need to try to get a
                     // signed assertion, and if it fails, move to the next credentials
-                    _= await signedAssertion!.GetSignedAssertion(CancellationToken.None);
```

Comment:
An Async method is supposed to end with Async and have a cancellation token as the last parameter? Maybe we should do that?

Reply:
oh I see the cancellation token is in the AssertionRequestOptions ... a bit strange, but works

Reply:
Agreed on the "Async" suffix.

It's a fair comment for Options object + CancellationToken. For keeping things simple with MSAL.NET, let's leave it as is ?